ci/pipeline.yml

```diff
@@ -0,0 +1,85 @@
+---
+jobs:
+- name: deploy-production
+  plan:
+  - get: cg-docs-source
+    trigger: true
+  - task: build
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: jojomi/hugo
+          tag: '0.16'
+      inputs:
+      - name: cg-docs-source
+      outputs:
+      - name: cg-docs-compiled
+      run:
+        dir: cg-docs-source
+        path: sh
+        args:
+        - -exc
+        - |
+          # install dependencies
+          # git required for NPM, the rest required for node-sass->node-gyp
+          apk add --no-cache --update g++ git make nodejs python
```

Comment:
So we actually don't need to compile anything with node-sass in this project, the build process really just copies files from cg-style into a public folder so it can be referenced? At what part of the build process were you getting errors about not having python?